src/component/TodoList.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import TodoItem from './TodoItem';
+import User from './User';
+import './TodoList.css';
+
+export default class TodoList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      todos: this.props.todos,
+      users: this.props.users,
+      sortBy: 'time'
+    }
+    this.sort = this.sort.bind(this);
+  }
+
+  sort(e) {
+    if (e.target.dataset.sortType === 'title') {

```

Comment:
try to write one function for sort and call it 3 times with different parameters.
and use `localeCompare`